### PR TITLE
Updating copyright year from 2021 to 2022

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Changing Dev docs footer banner copyright year
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* n/a

### CODE REVIEW

**Review by francoislehoux-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ module.exports = ctx => ({
     ],
 
     forum_url: 'https://devforum.okta.com/',
-    copyright_text: 'Copyright © 2021 Okta.'
+    copyright_text: 'Copyright © 2022 Okta.'
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:248]: Would we be against doing this instead? We wouldn't have to ever think about it again 😅 
```suggestion
    copyright_text: `Copyright © ${new Date().getUTCFullYear() || '2022'} Okta.`
```

@shawyu-okta

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:248]: LGTM 🚀 !

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:248]: Great idea. Testing locally right now.

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/.vuepress/config.js:248]: Looks good. Will commit the suggestion. Thanks @francoislehoux-okta

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**